tests/scanner.rs

```diff
@@ -75,59 +81,76 @@ fn flatten_diffs(diffs: &[Difference]) -> Vec<Difference> {
                 .map(Difference::Rem)
                 .collect::<Vec<_>>()
                 .into_iter(),
-        }).collect()
+        })
+        .collect()
+}
+
+fn run_codegen_test(generated_file_path: &Path, expected_output: &str) {
+    match Command::new("rustfmt")
+        .arg("--config-path")
+        .arg(env!("CARGO_MANIFEST_DIR"))
+        .arg(generated_file_path)
+        .status()
```

Comment:
To better handle rustfmt changes, maybe we could:

- in the scanner tests, run `rustfmt` on both the generated file and the target file
- run `rustfmt --chek` on the target files in the `FMT` travis build

Reply:
Why both? If we `rustfmt --check` the test assets in CI, we'll have to update them anyway if rustfmt changes, also running it on them without `--check` as part of the test code seems like it would have no additional benefit while making the code (slighly) longer.

Reply:
I agree it's not strictly needed. My intent was to ensure that only the rustfmt job would break if a rustfmt update changed the formatting.